modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -43,59 +49,131 @@ sourceFiles:
       storageClassName: image-registry-sc
       volumeMode: Filesystem
   # persistent volume
-  - fileName: ImageRegistryPV.yaml <3>
+  - fileName: ImageRegistryPV.yaml <2>
     policyName: "pv-for-image-registry"
     metadata:
       annotations:
-        ran.openshift.io/ztp-deploy-wave: "100"  <4>
+        ran.openshift.io/ztp-deploy-wave: "100"
   # image registry config
-  - fileName: ImageRegistryConfig.yaml <5>
+  - fileName: ImageRegistryConfig.yaml <3>
     policyName: "config-for-image-registry"
     complianceType: musthave <5>
     metadata:
       annotations:
-        ran.openshift.io/ztp-deploy-wave: "100" <6>
+        ran.openshift.io/ztp-deploy-wave: "100"
     spec:
       storage:
         pvc:
           claim: "image-registry-pvc"
 ----
-<1> Set the appropriate value for `ztp-deploy-wave` depending on whether you are configuring image registries at the site, common, or group level. `ztp-deploy-wave: "100"` is appropriate for an individual site.  ZTP deploy waves are used to order how policies are applied to the spoke cluster. All policies created by `PolicyGen` have a ztp deploy wave by default.
-<2> Set the appropriate value for `ztp-deploy-wave` as in note 1.
-<3> This assumes that `mount_point` is set to `/var/imageregistry` in `SiteConfig` using StorageClass `image-registry-sc` (see the topic on configuring disk partitioning with `SiteConfig`).
-<4> Set the appropriate value for `ztp-deploy-wave` as in note 1.
-<5> Configure registry to point to the PVC created above.
-<6> Set the appropriate value for `ztp-deploy-wave` as in note 1.
+<1> Set the appropriate value for `ztp-deploy-wave` depending on whether you are configuring image registries at the site, common, or group level. `ztp-deploy-wave: "100"` is appropriate for an individual site. ZTP deploy waves are used to order how policies are applied to the spoke cluster. All policies created by `PolicyGen` have a ztp deploy wave by default.
+<2> In `ImageRegistryConfig.yaml`, ensure that the `spec.local.path` field is set to `/var/imageregistry` to match the value set for the `mount_point` field in the `SiteConfig` CR.
+<3> Do not set `complianceType: mustonlyhave`. `mustonlyhave` causes the registry pod deployment to fail.
+
+. Create the `PolicyGenTemplate` CR:
++
+[source,terminal]
+----
+$ oc create -f example-sno-site.yaml
```

Comment:
This should be removed. Instead say something like "Sync ArgoCD". 

A note -- PGT and StieConfig only works with ZTP Containers (there's no operator in the cluster that understand the CRs) so ArgoCD/GitOps is an integral part of it.

Reply:
Oops yes. Updated.